index.js

```diff
@@ -0,0 +1,57 @@
+const omit = require('lodash/omit');
```

Comment:
:( https://github.com/lodash/lodash/issues/2930

Reply:
WTF, that's so unlike lodash to be so opinionated about that. I don't think when you're the defacto standard library of a language you should be able to tell everyone with there infinite use-cases that common pattern foo is not good.

Reply:
but computers...